client/webserver/locales/zh-cn.go

```diff
@@ -162,7 +162,9 @@ var ZhCN = map[string]string{
 	"Balance":                "余额", // alt. 平衡
 	"Actions":                "操作", // alt. 动作
 	"Restoration Seed":       "恢复种子",
+	"Restoration Seed Phrase": "Restoration Seed Phrase",
 	"Restore from seed":      "从种子恢复",
+	"Restore from seed phrase": "Restore from seed phrase",
```

Comment:
I didn't update/added translations, should I use `google translate` or invite someone to check it out ?

seems like overkill doing it every pull request, maybe better do it in batches (every so often invite someone to update translations) ?

Reply:
> seems like overkill doing it every pull request, maybe better do it in batches (every so often invite someone to update translations) ?

Yes, that's what I said previously.  This should not hold up dev work.

Reply:
so, I see 2 options here:
- leaving it English (until somebody can pick it up and translate properly)
- using google translate, and then someone will spot discrepancies and improve upon it

which approach do you usually follow ? I can see examples of [1st option being used](https://github.com/LasTshaMAN/dcrdex/blob/607675366f8116e01e05dd755883a4cec6940bc4/client/webserver/locales/pt-br.go#L32-L32), I guess this is the way to go

Reply:
Yeah, just leave it English. You can remove it from this file and other locale except the **en-us.go** locale. `go generate` will use the default value English when preparing the HTML template files.

Reply:
Yeah, just leave it English. You can remove it from this file and other locale except the **en-us.go** locale. `go generate` will use the default value English when preparing the HTML template files.

Reply:
> Yeah, just leave it English. You can remove it from this file and other locale except the en-us.go locale.

applied this